packages/exo/test/test-heap-classes.js

```diff
@@ -53,6 +53,18 @@ test('test defineExoClass', t => {
   });
   t.deepEqual(upCounter[GET_INTERFACE_GUARD](), UpCounterI);
   t.deepEqual(ownKeys(UpCounterI.methodGuards), ['incr']);
+  t.is(UpCounterI.symbolMethodGuards, undefined);
+
+  const symbolic = Symbol.for('symbolic');
+  const FooI = M.interface('Foo', {
```

Comment:
Please also add a test with an exo implementing FooI and then call the symbol-named method to test that the raw method is guarded by the methodGuard.

Reply:
Done.